### PR TITLE
Copenhagen estimates for HIV and HCV. Added multiplication method to PrevalenceAbsolute. Edited test.py, as it runs an example test on HIV estimates. 

### PR DESCRIPTION
**Pull request by simonleandergrimm**
Added Copenhagen estimates for HIV and HCV. 

Also edited test.py, as it runs an example test on HIV estimates.

Added multiplication method to PrevalenceAbsolute.

### CODE REVIEW

File: pathogens/hiv.py [hunk 1]
```diff
@@ -69,16 +101,43 @@ def estimate_prevalences() -> list[Prevalence]:
...
                 state="California", county="Los Angeles County", year=2020
             )
         )
-        * la_unsuppressed_fraction_2020,
+        * la_unsuppressed_fraction_2020
+    )
+    # Extrapolating HIV rate in Copenhagen backwards from 2022 to 2015-2018. We assume that HIV has mostly stayed constant.
+    copenhagen_2022 = (
+        denmark_infected_undiagnosed_2022.to_rate(copenhagen_population_2022)
+        * hiv_share_copenhagen
```
Comment (dp-rice) [inline pathogens/hiv.py:115]: I don't understand this calculation. It looks like you're taking the number of undiagnosed cases in all of Denmark and dividing by the population in all of Denmark to get the prevalence (of undiagnosed cases) in Denmark. Multiplying this by the fraction of cases that are in Copenhagen won't get you the prevalence in Copenhagen without also including the population of Copenhagen in the calculation.

File: pathogens/hiv.py [hunk 2]
```diff
@@ -58,6 +58,38 @@
...
+hiv_share_copenhagen = Scalar(
+    scalar=0.44,
+    source="https://doi.org/10.1080/14034940510005671",  # First paragraph in introduction
+    # The cumulative number of AIDS cases in Copenhagen represented 44% by the
+    # end of 1998, while the city’s population constituted only 9% of the
+    # total population in Denmark.
+    date="1998",  # The paper is from 2005, but the data is from 1998.
+    # Cases still cluster in Copenhagen nowadays, though the share is lower.
+    # E.g.,out of new tests in 2019 and 2020, 37% (45 / 143) and 19% of cases
+    # (28/108)
```
Comment (dp-rice) [inline pathogens/hiv.py:81]: Why use the older estimate if newer data is available? 1998 is a long time ago.

**Review by dp-rice — CHANGES_REQUESTED** [review]

File: pathogens/hiv.py (hunk 1 above)
Comment (simonleandergrimm) [reply pathogens/hiv.py:115]: The population I'm using is the population from Copenhagen, see definition of `copenhagen_population_2022`:
```
copenhagen_population_2022 = Population(
    people=652_221,
    country="Denmark",
    date="2022",
    source="https://kk.statistikbank.dk/statbank5a/SelectVarVal/Define.asp?MainTable=KKBEF1"
    # Search variables: Copenhagen in total, Gender in general, Age in total,
    # 2022 Q4
)
```

The order of the calculation is still kind of weird (going from country-wide absolute prevalence directly to Copenhagen prevalence rate, and then scaling down to account for the fact that half of infected people are outside of Copenhagen).

I've changed the PrevalenceAbsolute Python object to make the calculation clearer:

I take the overall, absolute number of undiagnosed HIV positive individuals (denmark_infected_undiagnosed_2022), and scale it down by the share of individuals with HIV in Copenhagen(hiv_share_copenhagen). 

Then, I turn the absolute number of HIV-positive undiagnosed people in Copenhagen (a PrevalenceAbsolute object) into a Prevalence, by incorporating Copenhagen's population.

Comment (simonleandergrimm) [reply pathogens/hiv.py:115]: The population I'm using is the population from Copenhagen, see definition of copenhagen_population_2022:

copenhagen_population_2022 = Population(
    people=652_221,
    country="Denmark",
    date="2022",
    source="https://kk.statistikbank.dk/statbank5a/SelectVarVal/Define.asp?MainTable=KKBEF1"
    # Search variables: Copenhagen in total, Gender in general, Age in total,
    # 2022 Q4
)
The order of the calculation is still kind of weird (going from country-wide absolute prevalence directly to Copenhagen prevalence rate, and then scaling down to account for the fact that half of infected people are outside of Copenhagen).

I've changed the PrevalenceAbsolute Python object to make the calculation clearer:

I take the overall, absolute number of undiagnosed HIV positive individuals (denmark_infected_undiagnosed_2022), and scale it down by the share of individuals with HIV in Copenhagen(hiv_share_copenhagen).

Then, I turn the absolute number of HIV-positive undiagnosed people in Copenhagen (a PrevalenceAbsolute object) into a Prevalence, by incorporating Copenhagen's population.

File: pathogens/hiv.py (hunk 2 above)
Comment (simonleandergrimm) [reply pathogens/hiv.py]: Good point, I changed it. Also added date from 2017 and 2018.

**Comment by simonleandergrimm** [comment]
Added multiplication method to PrevalenceAbsolute to make ` copenhagen_2022` HIV calculation more obvious

**Review by dp-rice — APPROVED** [review]
Looks good, thanks!

**Comment by dp-rice** [comment]
@simonleandergrimm I'm going to merge this so I can go ahead with the fitting

**Pull request merged**